charts/gateway-operator/templates/deployment.yaml

```diff
@@ -10,7 +10,11 @@ spec:
   replicas: {{ .Values.replicaCount }}
   selector:
     matchLabels:
+      {{- if .Values.selectorLabels }}
```

Comment:
Can we add this commented to the `values.yaml` so that people know that they can use it and know what setting this field means?